packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItem.jsx

```diff
@@ -117,6 +117,7 @@ class CollapsibleMenuViewItem extends React.Component {
     } = this.props;
 
     const { isCollapsibleGroupItem, isCollapsibleMenuItem } = this.context;
+    const { isSelectable, isSelectedButton } = customProps;
```

Comment:
I think we should have these as first-class props instead of custom props.

Reply:
We don't want to expose these props officially. Don't we want to keep it hidden through custom prop?

Reply:
These are Fusion specific props added to support passivity for fusion controls that are mapped to terra-collapsible-menu-view so we should not have them as first-class props !!